### ISSUE TITLE
LearningRate > 1?

### ISSUE THREAD

**Issue by young-chao** [deepinsight/insightface#2162]
When I run the [example](https://github.com/deepinsight/insightface/blob/master/recognition/arcface_torch/train.py) , I find that the output shows that the learning rate keeps increasing, even greater than 1. 

**python -m torch.distributed.launch --nproc_per_node=1 --nnodes=1 --node_rank=0 --master_addr="127.0.0.1" --master_port=12581 train.py configs/myconfig.py**
```
myconfig.py：
from easydict import EasyDict as edict
config = edict()
config.margin_list = (1.0, 0.5, 0.0)
config.network = "r50"
config.resume = False
config.output = None
config.embedding_size = 512
config.sample_rate = 1.0
config.fp16 = True
config.momentum = 0.9
config.weight_decay = 5e-4
config.batch_size = 128
config.lr = 0.1
config.verbose = 2000
config.dali = False
config.rec = "./data/DigiFace"
config.num_classes = 100 #93431
config.num_image = 500 # 5179510
config.num_epoch = 20
config.warmup_epoch = 0
config.val_targets = ['lfw', 'cfp_fp', "agedb_30"]
```

**Terminal output:**
```
Training: 2022-11-08 08:12:34,946-Speed 280.12 samples/sec   Loss nan   LearningRate 5.852250   Epoch: 0   Global Step: 520   Fp16 Grad Scale:  0   Required: -0 hours
Training: 2022-11-08 08:12:39,529-Speed 279.35 samples/sec   Loss nan   LearningRate 6.110028   Epoch: 0   Global Step: 530   Fp16 Grad Scale:  0   Required: -0 hours
Training: 2022-11-08 08:12:44,112-Speed 279.36 samples/sec   Loss nan   LearningRate 6.373361   Epoch: 0   Global Step: 540   Fp16 Grad Scale:  0   Required: -0 hours
Training: 2022-11-08 08:12:48,695-Speed 279.31 samples/sec   Loss nan   LearningRate 6.642250   Epoch: 0   Global Step: 550   Fp16 Grad Scale:  0   Required: -0 hours
Training: 2022-11-08 08:12:53,285-Speed 278.95 samples/sec   Loss nan   LearningRate 6.916694   Epoch: 0   Global Step: 560   Fp16 Grad Scale:  0   Required: -0 hours
Training: 2022-11-08 08:12:57,881-Speed 278.56 samples/sec   Loss nan   LearningRate 7.196694   Epoch: 0   Global Step: 570   Fp16 Grad Scale:  0   Required: -0 hours
Training: 2022-11-08 08:13:02,469-Speed 279.01 samples/sec   Loss nan   LearningRate 7.482250   Epoch: 0   Global Step: 580   Fp16 Grad Scale:  0   Required: -0 hours
Training: 2022-11-08 08:13:07,067-Speed 278.44 samples/sec   Loss nan   LearningRate 7.773361   Epoch: 0   Global Step: 590   Fp16 Grad Scale:  0   Required: -0 hours
Training: 2022-11-08 08:13:11,665-Speed 278.46 samples/sec   Loss nan   LearningRate 8.070028   Epoch: 0   Global Step: 600   Fp16 Grad Scale:  0   Required: -0 hours
Training: 2022-11-08 08:13:16,262-Speed 278.47 samples/sec   Loss nan   LearningRate 8.372250   Epoch: 0   Global Step: 610   Fp16 Grad Scale:  0   Required: -0 hours
Training: 2022-11-08 08:13:20,866-Speed 278.10 samples/sec   Loss nan   LearningRate 8.680028   Epoch: 0   Global Step: 620   Fp16 Grad Scale:  0   Required: -0 hours
Training: 2022-11-08 08:13:25,464-Speed 278.43 samples/sec   Loss nan   LearningRate 8.993361   Epoch: 0   Global Step: 630   Fp16 Grad Scale:  0   Required: -0 hours
Training: 2022-11-08 08:13:30,061-Speed 278.47 samples/sec   Loss nan   LearningRate 9.312250   Epoch: 0   Global Step: 640   Fp16 Grad Scale:  0   Required: -0 hours
Training: 2022-11-08 08:13:34,664-Speed 278.11 samples/sec   Loss nan   LearningRate 9.636694   Epoch: 0   Global Step: 650   Fp16 Grad Scale:  0   Required: -0 hours
```
